### PR TITLE
Space-Travelers-Hub app

### PR DESCRIPTION
**Pull request by AleksandarIvezic**
## Changes:
- Created React app Space-Travelers-Hub
- Created rockets and missions components
- Fetched data from external APIs
- Used Redux for storing rockets and missions state
- Styled using react-bootstrap

### CODE REVIEW

**Review by mke2111 — CHANGES_REQUESTED** [review]
### Hey @ShinobiWarior & @TedLivist,

**This is a great project you have going on, well-done lads**. 
You have done a splendid job with this project, the implementation covers most of the required concepts. Cheers! 🥇 👍🏽 
   
**However, there are a few sections to improve before your project is approved.** 🔴 
<hr>

## Required Changes ♻️
_Check the inline comments under the review for sections to improve._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Missions.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+                />
+              </tr>
+            ))}
+          </tbody>
+        </table>
+        {/* {missions && missions.map((mission) => (
+          <li key={mission.mission_id}>
+            <OneMission name={mission.mission_name} />
+          </li>
+        ))} */}
```
Comment (mke2111) [inline src/components/Missions.js]: - [x] Clean up your file. It is a standard to remove all commented-out code snippets.

File: src/components/App.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+          <Route path="/my-profile">
+            <MyProfile />
+          </Route>
+        </Switch>
+      </div>
+    </Provider>
+  );
+}
+
+export default App;
```
Comment (mke2111) [inline src/components/App.js]: - [x] You are missing the most important part of ensuring the project will work as expected. Write unit tests with React Testing Library and Jest (for unit tests). Cater for all functions. Remember to include snapshot tests.

File: src/components/MyProfile.js [hunk 3]
```diff
@@ -0,0 +1,56 @@
...
+                className="list-group-item"
+              >
+                {rocket.name}
+              </li>
+            ))}
+          </ul>
+        )}
+        {(!reservedRockets || !reservedRockets[0]) && (
+          <h4 className="text-danger m-4">No reserved rockets! :(</h4>
+        )}
```
Comment (mke2111) [inline src/components/MyProfile.js:50]: OPTIONAL

- The My Profile section can be enhanced by adding a placeholder message when the "My Missions" or "My Rockets" lists are empty (no missions joined or no rockets reserved).
- The My Profile section can be enhanced by adding the "Cancel reservation" and "Leave Mission" buttons to the lists here. Clicking them should dispatch the actions you have already used in the main Rockets and Missions sections.
- The My Profile section can be enhanced by adding the "Read more" button for each mission and rocket. Upon click, it should open a corresponding Wikipedia page in a new tab. NOTE - you need to get that extra Wikipedia URL from the API's payload.

File: src/components/App.js (hunk 2 above)
Comment (Emmyn5600) [reply src/components/App.js]: Well done

File: src/components/Missions.js (hunk 1 above)
Comment (Emmyn5600) [reply src/components/Missions.js]: Well done

**Review by Emmyn5600 — APPROVED** [review]
Hi,

Good job so far! 👏🏼 @ShinobiWarior and @TedLivist 


## STATUS: APPROVED :muscle::clap:

Congratulations! :tada:
Your project is complete!  It's time to merge it :shipit:
Great work on implementing the changes requested by the previous reviewer.

 #### Optional suggestions
 
 - N/A.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding! :clap: :clap: :clap:

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

#### _**Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.**_
------------------------------------------------------------------------------------------------------------------------------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using this [form.](https://airtable.com/shrQAqnBwek5a0O0s)

**Pull request merged**